user/ignis/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "ignis"
+pkgver = "0.3"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "gobject-introspection",
+    "pkgconf",
+    "git",
+]
+makedepends = [
+    "gtk4-devel",
+    "gtk4-layer-shell-devel",
+    "libpulse-devel",
+]
+depends = [
+    "gtk4-layer-shell-devel",
+    "libpulse",
```

Comment:
this is autotraced

Reply:
also, the build results in a `libgvc.so` being dumped into `/usr/lib` which seems wrong because that's a private library

Reply:
with autotraced you mean that it is already pulled in by other dependencies?

Reply:
by autotraced i mean that cbuild checks executables/shared libs for their runtime dependencies and knows about the libpulse dependency already